packages/cli/src/commands/setup/auth/templates/auth.js.template

```diff
@@ -1,4 +1,4 @@
-import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/api'
+import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/graphql-server'
```

Comment:
parseJWT is still in @redwoodjs/api I believe

Reply:
Whoops. You are correct. Thanks!